cmd/web/webapp.go

```diff
@@ -35,15 +33,10 @@ func NewWebappCmd() *cobra.Command {
 }
 
 func serve(cmd *cobra.Command, args []string) {
-	engine := web.NewEngine()
-
-	s := &http.Server{
-		Addr:           fmt.Sprintf("%s:%d", host, port),
-		Handler:        engine,
-		ReadTimeout:    10 * time.Second,
-		WriteTimeout:   10 * time.Second,
-		MaxHeaderBytes: 1 << 20,
-	}
```

Comment:
This configuration was here for a reason: default timeouts and header limit of the go http server are not ideal.